packages/boiler-config-webpack/src/plugins.js

```diff
@@ -40,6 +44,22 @@ export default function(opts) {
     toolsPlugin
   ];
 
+  if (!isDev && (isMainTask || isGlobalTask)) {
+    plugins.push(
+      new SriStatsPlugin({
+        algorithm: 'sha512',
+        allow: /\.(js|css)$/i,
+        customStatsKey: 'rails',
```

Comment:
Does this key matter?  (I'm guessing not in the context of this build process, but just curious!)